plugins/SitesManager/tests/UI/EmptySite_spec.js

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Piwik - free/libre analytics platform
+ *
+ * Screenshot integration tests.
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+describe("EmptySite", function () {
+    this.timeout(0);
+
+    var generalParams = 'idSite=4&period=day&date=2010-01-03';
+
+    before(function () {
+        testEnvironment.pluginsToLoad = ['SitesManager'];
```

Comment:
Ah OK I will remove that line then, I just used the UI tests generator and that was added automatically.